HW06.Rmd

```diff
@@ -19,15 +19,76 @@ Steps:
 
 From the RNA-Seq files, you only need the "Name" from one file and the "TPM" column from all the files. TPM stands for "transcripts per million" and is a common unit for normalized expression data. 
 ```{r load in data}
+library("readr")
+library("dplyr")
+library("tidyr")
+library("ggplot2")
 #hint, using apply (specifically sapply) you can read in the data into a list and then bind the columns together. Or you can use a for loop too. 
 
-#you only need the 
+H3K27M_filepath_list <- list.files(path = "RNA_Seq_processed/H3K27M", full.names = TRUE)
+WT_filepath_list <- list.files(path = "RNA_Seq_processed/WT", full.names = TRUE) #list.files() simply creates a list containing the file names of all the files in a folder. With the full.names argument set to TRUE, the function outputs the path of the file instead of just the file name

```

Comment:
You could make a single list with recursive = TRUE, but you would need to add if statements to the for loops below to distinguish between H3K27M and WT.
I think it would condense the code quite a lot